src/base-config/keyboard.json

```diff
@@ -71,7 +71,13 @@
         }
     ],
     "edit.splitSelIntoLines": [
-        "Ctrl-Alt-L"
+        {
+           "key": "Ctrl-Alt-L"
```

Comment:
@Projjol I think he meant change the shortcut only for linux, which sounds ok to me. I have seen different key bindings for the same command on different platforms in other cases as well (Moving line up / down). So this change should be fine